docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -45,7 +45,7 @@ SHOW TABLE EXTENDED [IN|FROM database_name] LIKE 'identifier_with_wildcards' [PA
     Specifies the regular expression pattern that is used to filter out unwanted tables.
     <ul> 
        <li> Except for `*` and `|` character, the pattern works like a regex.</li>
-       <li> `*` alone matches 0 or more characters and `|` is used to separate multiple different regexes,
+       <li> `*` alone matches 0 or more characters and `|` is used to separate multiple regex,
```

Comment:
Isn't `regexes` fine?

Reply:
@HyukjinKwon I think singular or plural it is regex,not regexes.

Reply:
Hm, I'd say regexes

Reply:
O thanks